h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -238,7 +245,19 @@ public void compute2() {
         Scope.exit();
       }
       tryComplete();
+      if (_modelBuilderListener != null) {
+        _modelBuilderListener.onModelSuccess(_job.get());
```

Comment:
What will happen in the case when the MB is running nested in another job (`trainModelNested`)? Is there another way to access the model instance?

Reply:
Got inspired by `get` method from this very `ModelBuilder`. - `  public final M get() { assert _job._result == _result; return _job.get(); }`

I guess I can use `_result.get()` (pushed).